Josh-TD/SDM-ISO
# Orm models

Finished Models for the ORM
we can now read/write etc using the models in the backend and create the stuff for the API calls

the models are linked to the database information with rows and all 
no rows are missing or errors called, but technically i havent looked at every table to make sure 

to test the models/database:
We now have a database where we can run any experiments on 
its in the same AWS instance, a database named "sdm-test-db"

make sure you have an UPDATED env.properties that looks like this:
`DB_URL=jdbc:postgresql://database-1.cra2rjnhxmun.us-east-2.rds.amazonaws.com:5432/sdm-test-db
DB_USERNAME=postgres
DB_PASSWORD=sdm-iso559`

Comment out the stuff in application.properties 
run the script in discord on the database
then `mvn spring-boot:run`
